front/app/utils/configs/votingMethodConfig/messages.ts

```diff
@@ -78,11 +78,6 @@ export default defineMessages({
     id: 'app.utils.votingMethodUtils.midSentenceVotes',
     defaultMessage: 'votes',
   },
-  numberOfVotes: {
-    id: 'app.utils.votingMethodUtils.numberOfVotes',
-    defaultMessage:
-      '{numberOfVotes} {numberOfVotes, plural, =0 {{votesTerm}} one {{voteTerm}} other {{votesTerm}}}',
-  },
```

Comment:
Wrote a comment here, but never mind.

Reply:
The still relevant part: I maybe overlooked that if we put it in the code, there's no chance to move things around for languages that have use different order in one way or another.

Reply:
Hm Luuc says that Crowdin doesn't allow to change the order of the words. But anyway, I believe this issue is rather minor, so either closing this PR or merging is fine to me :smile: